code/_onclick/hud/fullscreen.dm

```diff
@@ -7,15 +7,28 @@
 		screen = screens[category]
 		if(screen.type != type)
 			clear_fullscreen(category, FALSE)
-			return .()
```

Comment:
.() does indeed make the proc call itself (NOT the parent).

Read the byond documentation sometime.